libasn1compiler/asn1c_save.c

```diff
@@ -294,6 +294,58 @@ asn1c__save_example_am_makefile(arg_t *arg, const asn1c_dep_chainset *deps, cons
 	return 0;
 }
 
+static int
+asn1c__save_autotools_example(const char *destdir,
+                              const char *program_makefile_name) {
+	FILE *fd;
+	const char* confac = "configure.ac";
+	const char* makeam = "Makefile.am";
+
+	if ((access(confac, F_OK) != -1)
+	    || (access(makeam, F_OK) != -1))
+	{
+		safe_fprintf(stderr, "Refusing to overwrite existing '%s' or '%s'\n", confac, makeam);
+		return -1;
+	}
+
+	fd = asn1c_open_file("", confac, "", 0);
+	if(fd == NULL) {
+		perror(confac);
+		return -1;
+	}
+
+	safe_fprintf(fd,
+	             "AC_INIT([asn1covert],[0.1])\n"
```

Comment:
Convert.

Reply:
Sorry, that was left over from my previous PR. (I thought `asn1convert` was a better name than `convert-example`). I'll change it to `AC_INIT([convert-example],[0.1])` if you'd prefer that

Reply:
Oh... maybe you were referring to the typo instead :)
asn1covert => asn1convert

Reply:
Yes, I was referring to the typo.